### ISSUE TITLE
Register bcgen on PECL

### ISSUE THREAD

**Issue by vjardin** [nanosonde/bcgen#9]
Work in progress https://pecl.php.net/package/bcgen

**Comment by nanosonde** [comment]
@vjardin 

I would like to remove this repo completely as I do not have the time and motiviation anymore to work on this.
Originally I have created BCgen to convince another developer of a partly-closed-source PHP project to move to PHP7 as he is using bcompiler which is only supported until PHP5.3.
Sadly, I never even got a reply from him for my efforts.

Do you want to take a copy of this?
You could then host the project and the PECL package yourself.

**Comment by vjardin** [comment]
OK, forked: https://github.com/vjardin/bcgen
Thanks, I believe it is a nice initiative ; we'll continue so it can become a PHP extension. I have already seen some people moving away from bcompiler to use bcgen... so you did the right move.

**Comment by nanosonde** [comment]
@vjardin 
Good to hear. :-)

Concerning the fork:
Wouldn't it be better if you delete your fork and I transfer the ownership of this repo to you?
This way the issues are also kept. I have already tried to initiate the process here in github, but it complains taht you already have a repo named "bcgen".

What do you think?

NB: You should also update the bcgen PECL page to the forked version.

**Comment by nanosonde** [comment]
And the original bcompiler author Alan has also updated the link on the bcompiler PECL page to point to the bcgen PECL page.

**Comment by vjardin** [comment]
OK, I'll remove the fork so you can transfer it and we can keep the records of the background. I did not know about it. Please, check in about 1h from now ; the issue should be fixed.

**Comment by nanosonde** [comment]
It seems to have worked. :-)

**Comment by nanosonde** [comment]
Do not forget to update the PECL page with new link.

**Comment by vjardin** [comment]
Yes. Thanks for the work you initiated. I hope to see you again into the PHP community.

**Comment by 4r7if3x** [comment]
@vjardin That would be great if you can make releases available at PECL and also provide an automated encoding script along side detailed instructions. That would help more developers start using that and also web hosting providers feel fine to install it on their servers.

**Comment by vjardin** [comment]
@SamAriafar
Stay tuned. I'll do it. First I need the CI to be working. I'll be at Fosdem tomorrow. I'll be happy to chat about it.

**Comment by benyaminl** [comment]
> @SamAriafar
> Stay tuned. I'll do it. First I need the CI to be working. I'll be at Fosdem tomorrow. I'll be happy to chat about it.

I want to support via patreon if it's possible, monthly.

**Comment by eoksum** [comment]
@vjardin Hi there, there are many issues open on this repo. Can you please check them?

**Comment by vjardin** [comment]
I went thru them yesterday. Thanks for the ping.
